plugins/web-plugin/src/main/java/ru/sbtqa/tag/pagefactory/web/utils/WebExpectedConditionsUtils.java

```diff
@@ -92,33 +93,45 @@ public static void waitForTextInInputExists(WebElement webElement, long timeout)
     /**
      * Wait until specified text either appears, or disappears from page source
      *
-     * @param text                text to search in page source
+     * @param text text to search in page source
      * @param shouldTextBePresent boolean, self explanatory
      * @throws ru.sbtqa.tag.pagefactory.exceptions.WaitException TODO
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)